src/scripts/init.ts

```diff
@@ -147,30 +153,36 @@ async function saveMqttTeamDeviceDataToSSM(teamDevice: MqttTeamDevice) {
     await SSM.send(new PutParameterCommand({
         Name: NRFCLOUD_CLIENT_CERT_PARAM,
         Value: teamDevice.clientCert,
-        Type: ParameterType.STRING
+        Type: ParameterType.STRING,
+        Overwrite: true
```

Comment:
Need these `Overwrite` params so the little-known `--reset` flag would work (apparently never tested until now).